docs/pages/use-teleport/teleport-connect.mdx

```diff
@@ -0,0 +1,108 @@
+version of Teleport Connect. As of now, Teleport Connect supports macOS only. Double-click the
+downloaded .dmg file and drag the Teleport Connect icon to the Applications folder.
+
+To upgrade Teleport Connect to a newer version, drag the new version to the Applications folder.
+
+## Interface Overview
+
+![interface overview of Teleport Connect](../../img/use-teleport/connect-ui-overview.png)
+
+The top bar of Teleport Connect consists of:
+
+- The profile selector (the top right) allowing you to switch between profiles on different Teleport
+  clusters as well as log in or out of the clusters.
+- The connection list (the top left) showing recent connections, allowing you to seamlessly switch
+  between them.
+- The command bar (in the middle) which launches any command you input in a new terminal tab.
+- The cluster selector (to the left of the command bar) shows up only if there are leaf clusters
+  connected to the root cluster. It lets you browse leaf cluster resources. It also changes
+  which cluster the commands in the command bar are going to target.
+
+## Connecting to an SSH Server
+
+1. Open a tab with cluster resources by clicking on the plus symbol at the right end of the tab bar.
+   You can also press `Cmd + T` to achieve the same result.
+2. Look for the SSH server you want to connect to and click the Connect button to the right.
+3. Select or enter the SSH user you wish to log in as and press `Enter`.
+4. A new tab will open with a shell session on the chosen server.
+
+Alternatively, you can type `tsh ssh` into the command bar and the autocompletion will help you
+connect to a server.
+
+## Connecting to a Database
+
+1. Open a tab with cluster resources by clicking on the plus symbol at the end of the tab bar. You
+   can also press `Cmd + T` to achieve the same result.
+2. Select the Databases section.
+3. Look for the database server you wish to connect to and click the Connect button to the right.
+4. Select or enter the database user you with to use and press `Enter`.
+5. A new tab will open with a new connection established between your device and the database
+   server.
+
+This connection will remain active until you click the Close Connection button or close Teleport
+Connect. The port number will persist between app restarts – you can set up your favorite client
+without worrying about the port suddenly changing.
+
+### With a GUI Client
+
+To connect with a GUI client, follow the instructions in the database connection tab under the
+Connect with GUI section.
+
+### With a CLI Client
+
+The database connection tab shows the command that can be used to connect to the database. You can
+modify the database name of the connection and then click the Run button to open a new
+terminal tab with that command executed.
+
+## Connecting to Multiple Clusters
+
+After you add your first cluster, you can keep adding more by opening the profile selector in the
```

Comment:
What does "After you add your first cluster" mean? Add your first leaf cluster to your root cluster? Or does this relate to the UI?

Reply:
> What does "After you add your first cluster" mean? Add your first leaf cluster to your root cluster? Or does this relate to the UI?

What it really refers to is logging in to a cluster. Tbh our UI right now says "Add another cluster" but what it really should say is just "Log in to another cluster". I'll update this part along with the UI.

Reply:
Actually, I'll change just the docs for now.